aspnetcore/fundamentals/index.md

```diff
@@ -16,11 +16,12 @@ This article is an overview of key topics for understanding how to develop ASP.N
 
 The `Startup` class is where:
 
-* Any services required by the app are configured.
+* Services required by the app are configured.
 * The request handling pipeline is defined.
 
-* Code to configure (or *register*) services is added to the `Startup.ConfigureServices` method. *Services* are components that are used by the app. For example, an Entity Framework Core context object is a service.
-* Code to configure the request handling pipeline is added to the `Startup.Configure` method. The pipeline is composed as a series of *middleware* components. For example, a middleware might handle requests for static files or redirect HTTP requests to HTTPS. Each middleware performs asynchronous operations on an `HttpContext` and then either invokes the next middleware in the pipeline or terminates the request.
+*Services* are components that are used by the app. For example, an Entity Framework Core context object is a service. Code to configure (or *register*) services is added to the `Startup.ConfigureServices` method.
```

Comment:
```suggestion
*Services* are components that are used by the app. Code to configure (or *register*) services is added to the `Startup.ConfigureServices` method.
```

Alternatively, is there something more general/generic that could be used for the service example? Even something like MVC, which is widely understood as an architectural pattern provided by an app framework.

Reply:
> is there something more general/generic that could be used for the service example?

MVC could be misleading, as it sounds like a pattern than a component.  I'm open to other ideas ... I could refer to a logger, that would be more general than EF.

Reply:
Just a passing thought. EF is probably a cryptic tech for some readers. I like the logger idea.